### PR TITLE
Terms & Conditions screen

### PR DESCRIPTION
**Pull request by VictoriaAjala**
This fixes issue #3 

Screenshot from 2023-01-14 17-55-39

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -50,6 +51,8 @@ function App() {
         <Route path="/payment" element={<PaymentMethod />} />
         <Route path="/place-order" element={<Payment />} />
         <Route path="/order/:_id" element={<OrderDetails />} />
+        <Route path="/Terms" element={<TermsCondition />} />
```
Comment (ifedayoagboola) [inline src/App.js:54]: All route paths should be in small letters

File: src/components/Footer.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ const Footer = () => {
                   <a href="" className="text-base text-gray-500 hover:text-gray-900 block">
                     Return Policy
                   </a>
-                  <a href="" className="text-base text-gray-500 hover:text-gray-900 block">
+                  <a href="/Terms" className="text-base text-gray-500 hover:text-gray-900 block">
```
Comment (ifedayoagboola) [inline src/components/Footer.js:47]: "/terms"

**Review by ifedayoagboola — APPROVED** [review]
I'll approve but don't forget to revisit the path thing. Following same convention is important!
Nice job!

**Comment by VictoriaAjala** [comment]
> I'll approve but don't forget to revisit the path thing. Following same convention is important! Nice job!

Right, I'll do that. Thank you!

**Pull request merged**